### ISSUE TITLE
Fix bug blocking the addition of multiple tags

### ISSUE THREAD

**Issue by lnrd96** [amosproj/amos2022ws04-specitem-database#133]
## User story
1. As a user I want to add several tags to an item. 
2. I want to be able delete tags.

## Acceptance criteria
Several tags can be added by and by.
A specific tag can be deleted.
These actions are possible on all versions of an item. 

*Note:*
The questions whether an old item that changes its tag should receive a new version will be answered on friday https://docs.google.com/document/d/1y1VdDOkj_gLkftg0uH9UBA54rtTJA35-kLW_yp6gGQQ/edit

### Notes
The functionality is suspected to be blocked by the following issues:
- empty commit value: when creating new version, because of an added tag
- `null` commit value needs to be handled
- changing an item by either adding a tag or by uploading a document should be differentiated

## Definition of done (DoD)
Feature has been fully implemented
No known errors
All acceptance criteria were met
Test passes
Peer code review passed

## Sprint Release DoD
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Peer code review passed
* Developers agreed to release
* Documentation updated
* Bill of materials updated
